src/preprocessors/vue.ts

```diff
@@ -48,8 +54,8 @@ export function vuePreprocessor(code: string, options: PrettierOptions) {
             console.warn(
                 '[@ianvs/prettier-plugin-sort-imports]: Could not process .vue file.  Please be sure that "@vue/compiler-sfc" is installed in your project.',
             );
-            throw err;
         }
+        throw err;
```

Comment:
Is it appropriate to always rethrow the error? In what cases do we not want to log the warning but still rethrow?

Reply:
The warning is an attempt to help guide the user towards a solution.  It detects when there is a `MODULE_NOT_FOUND`, and takes a guess that it's probably a missing dependency, so it gives that tip.  

I think it was just a mistake that it swallowed the error in the past, rather than re-throwing it.